packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -29,7 +32,27 @@ import { Alert } from '@patternfly/react-core';
 </React.Fragment>
 ```
 
-### Variations
+If no `variant` property is specified, then the variant will be set to "default". Otherwise, use the following variants:
+
+| Variant | Description |
+|---|---|
+| Default | Use for generic messages with no associated severity |
+| Info | Use for general informational messages |
+| Success | Use to indicate that a task or process has been completed successfully |
+| Warning | Use to indicate that a non-critical error has occurred|
+| Danger | Use to indicate that a critical or blocking error has occurred |
+
+
+### Alert variations
+
+PatternFly supports several properties and variations that can be used to add extra content to an alert. 
+
+* Use the `actionLinks` property to add one or more `AlertActionLink` components that place links beneath the alert message. You must pass in `href` and `component="a"` properties to have an `AlertActionLink` act as a proper link, rather than as a button.
+
+* Use a native HTML `<a>` element to add links within an alert message.
+
+* Use the `actionClose` property to add an `AlertActionCloseButton` component, which can be used to manage and customize alert dismissals.
+
```

Comment:
I'd add an additional bullet here since we demonstrate manipulating the headingLevel of the alert in the last couple alerts in the following example.
For accessibility reasons, it's recommended that people nest headings by their level and avoid skipping heading ranks when possible since it can be confusing (Make sure that a `h2` is not followed directly by an `h4`, for example)